crates/hyperqueue/src/client/commands/job.rs

```diff
@@ -161,6 +161,37 @@ pub async fn output_job_tasks(
     Ok(())
 }
 
+pub async fn output_jobs_tasks(
+    gsettings: &GlobalSettings,
+    connection: &mut ClientConnection,
+    job_id_selector: IdSelector,
+    task_selector: Option<TaskSelector>,
+) -> anyhow::Result<()> {
+    let message = FromClientMessage::JobDetail(JobDetailRequest {
+        job_id_selector,
+        task_selector,
+    });
+    let responses =
+        rpc_call!(connection, message, ToClientMessage::JobDetailResponse(r) => r).await?;
+
+    let jobs = responses
+        .into_iter()
+        .filter(|(job_id, opt_job)| match opt_job {
```

Comment:
You can use `.filter_map` to handle filter and map in one go. You can then also get rid of the `unwrap`.